contracts/token/psp22/tests.rs

```diff
@@ -291,4 +297,34 @@ mod tests {
 
         psp22.transfer_from(accounts.alice, accounts.eve, alice_balance + 1, Vec::<u8>::new());
     }
+
+    /// The mint was applied.
+    #[ink::test]
+    fn mint_extension_works() {
+        let mut psp22 = PSP22Struct::new(100);
+        let old_supply = psp22.total_supply();
+        psp22.mint(old_supply);
```

Comment:
Better to mint some another value and validate that old_supply + this value is current supply